src/main/java/me/jellysquid/mods/sodium/client/gui/SodiumOptionsGUI.java

```diff
@@ -17,24 +32,15 @@
 import net.minecraft.util.Formatting;
 import net.minecraft.util.Language;
 import net.minecraft.util.Util;
-import org.lwjgl.glfw.GLFW;
-
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.EnumSet;
-import java.util.HashSet;
-import java.util.List;
-import java.util.stream.Stream;
 
 public class SodiumOptionsGUI extends Screen {
-    private final List<OptionPage> pages = new ArrayList<>();
+    private final List<OptionTab> tabs = new ArrayList<>();
```

Comment:
Why are these and similar changes necessary?

Reply:
I was conceptually thinking that a "page" is the settings listed in a "tab", kind of how a page in a web browser is encapsulated within the tab.
I also needed to differentiate between an entry which held settings or acted as a button, and going with the word page would mean the subclasses would be named `OptionPageButton` and `OptionPagePage`.